django/http/multipartparser.py

```diff
@@ -321,6 +321,23 @@ def sanitize_file_name(self, file_name):
         file_name = file_name.rsplit('/')[-1]
         file_name = file_name.rsplit('\\')[-1]
 
+        forbidden_chars = { '>', '<', ':', '\"', '/', '\\', '|', '?', '*'} | set(chr(ch) for ch in range(32))
+        for char in forbidden_chars:
+            file_name = file_name.replace(char, '')
+
+        reserved_file_names =  {
+            'CON', 'PRN', 'AUX', 'NUL', 'COM1', 'COM2', 'COM3', 'COM4', 
+            'COM5', 'COM6', 'COM7', 'COM8', 'COM9', 'LPT1', 'LPT2', 
+            'LPT3', 'LPT4', 'LPT5', 'LPT6', 'LPT7', 'LPT8', 'LPT9' 
+        }
+
+        if file_name and file_name.split('.')[0].upper() in reserved_file_names:
+            file_name = file_name.replace(file_name.split('.')[0], '')
+        
+        extension = file_name.split('.')[-1]
+        for ext in ('..' + extension, ' .' + extension):
+            file_name = file_name.replace(ext, f'.{extension}')
```

Comment:
@vishalpandeyvip Have you missed my [comment](https://github.com/django/django/pull/14834#issuecomment-952762664)? IMO this goes too far and is unnecessarily complicated, we don't need `reserved_file_names` or hard-coded list of forbidden characters.

Reply:
I agree. I am removing these extra codes. I was trying to make changes as suggested in this [https://github.com/django/django/pull/14834#issuecomment-915974146](comment), In Microsoft documentation, there are some specific sets of reserved filenames and forbidden_characters, that's why I hardcoded them.